client/src/entrypoints/admin/page-editor.js

```diff
@@ -9,21 +10,21 @@ function InlinePanel(opts) {
   self.initChildControls = function (prefix) {
     const childId = 'inline_child_' + prefix;
     const deleteInputId = 'id_' + prefix + '-DELETE';
+    const currentChild = $('#' + childId);
+    const $up = currentChild.find('[data-inline-panel-child-move-up]');
```

Comment:
Just checking - has this and other JS changes in this file been tested with nested inline panels. It looks like some selectors could be a bit too greedy and grab child panels also. 

Note - RFC 78 will make this kind of thing easier as each controller will have its own scope (even for children with the same controller) .

However - for now it might be good to double check this.

Reply:
I have _not_ tested this with nested InlinePanel! I have some awareness of it but have not seen it set up in ages, and am unclear whether it’s a happy accident or a supported feature (#5919).